trains/community/gramps-web/app_versions.json

```diff
@@ -56,7 +56,7 @@
             ],
             "title": "Gramps Web",
             "train": "community",
-            "version": "1.1.7"
```

Comment:
please dont touch files under `trains/` directory

Reply:
Got it, sorry.